stacker/config/translators/kms.py

```diff
@@ -23,9 +23,24 @@ def kms_simple_decrypt(value):
         # In stacker we would reference the encrypted value like:
         conf_key: !kms us-east-1@CiD6bC8t2Y<...encrypted blob...>
 
-        # The above would resolve to
+        You can optionally store the encrypted value in a file, ie:
+
+        kms_value.txt
+        us-east-1@CiD6bC8t2Y<...encrypted blob...>
+
+        and reference it within stacker:
+
+        conf_key: !kms file://kms_value.txt
```

Comment:
the one confusing thing here is this path is relative to where you run the stacker function -- not where the conf file is located.

i'm not sure we can easily get the location of the conf file here